### PR TITLE
Persist session data

### PR DESCRIPTION
**Pull request by OscarSpruit**
- Pass session data around and persist it in the ViewModel

Bonus:
- Updated Mockito version and added Mockito Kotlin and Mockito Inline
- Added Turbine for testing flows

Thinking about migrating to JUnit 5 as well, because it makes tests so much prettier, but that is for another time

### CODE REVIEW

**Review by ozgur00 — APPROVED** [review]

File: drop-in/src/main/java/com/adyen/checkout/dropin/ui/viewmodel/DropInViewModel.kt [hunk 1]
```diff
@@ -168,6 +168,18 @@ class DropInViewModel(
         }
     }
 
+    fun onDropInServiceConnected() {
+        session?.let { session ->
```
Comment (jreij) [inline drop-in/src/main/java/com/adyen/checkout/dropin/ui/viewmodel/DropInViewModel.kt:172]: Can we add a log in case the session is null? I'm afraid we might miss out on any potential bugs if this method is called at the wrong time. A bit too late now but maybe we can do this in another PR.
